xbmc/TextureCache.cpp

```diff
@@ -433,6 +433,12 @@ bool CTextureCache::CleanAllUnusedImagesJob(CGUIDialogProgress* progress)
 
 void CTextureCache::CleanTimer()
 {
+  if (IsSleeping())
+  {
+    CLog::LogF(LOGDEBUG, "Texture cleanup postponed. System is sleeping.");
+    return;
```

Comment:
Cheers.

This timer is restarted after each run, returning here seems to stop the background process altogether. This suggested change will give it another chance later.

```suggestion
    m_cleanTimer.Start(std::chrono::hours(1));
    return;
```

Reply:
Good catch. Thanks. Will be changed.